autorest/azure/async.go

```diff
@@ -274,6 +277,10 @@ type pollingTracker interface {
 
 	// methods common to all trackers
 
+	// initializes a tracker's polling URL and method, called for each iteration.
+	// these values can be overridden by each polling tracker as required.
+	initHeaders() error
```

Comment:
>initHeaders [](start = 1, length = 11)

isn't this more like init form headers ?

Reply:
Yep the naming isn't the best but since we already have ```updateHeaders{}``` I wanted to keep this matching as the two are kinda related.

Reply:
eh these are not exported, can we rename them ?

Reply:
Good point, fixed.